### ISSUE TITLE
Isle of Wight Council request

### ISSUE THREAD

**Issue by tomqman** [robbrad/UKBinCollectionData#1011]
### Name of Council

Isle of wight council

### Example Address/Postcode

PO33 3PW

### Additional Information

https://www.iow.gov.uk/wasteday

### Verification

- [X] I''ve checked the [wiki](https://github.com/robbrad/UKBinCollectionData/wiki/Councils#contents) and verified that my council has not been added
- [X] I''ve checked that a request for my council does not already exist in the [Issues tracker](https://github.com/robbrad/UKBinCollectionData/issues?q=is%3Aopen+is%3Aissue+label%3A"council+request")
- [X] I have provided a tested working address/postcode/UPRN with bin collections available, as well as a link to the council''s website
- [X] I understand that this project is run by volunteer contributors and completion depends on numerous factors - even with a request, we cannot guarantee if/when your council will get a script

**Comment by jf-uu** [comment]
@robbrad does this need reopening? ac6e7d6 closes #1023 (as per https://github.com/robbrad/UKBinCollectionData/pull/1198#issuecomment-2620941184).

**Comment by robbrad** [comment]
Is it still not working?

@geekball provided a PR to
Fix this which was merged last night

Hence the closure of the issue

**Comment by jf-uu** [comment]
@geekball's PR (#1198) was for Herefordshire, they just mentioned the wrong issue in the commit message from what I can tell.

**Comment by robbrad** [comment]
Sorry I missed this

System auto closed it

**Comment by geekball** [comment]
> Sorry I missed this
> 
> System auto closed it

Yeah, sorry about that, I'd hoped by fixing the title it would've removed the mention, but no. 👎

**Comment by robbrad** [comment]
Hard to parse this one

![Image](https://github.com/user-attachments/assets/5cd22056-70ed-4750-85c8-a5b088563ede)

I think given there are onlky a handful of calendars I would use a Google Calendar with public ICS file 

https://github.com/robbrad/UKBinCollectionData/wiki/Councils#google-calendar-public

**Comment by robbrad** [comment]
Wont impliment due to PDFs - parsing PDFs is very unreliable - use Google calander approach instead

https://github.com/robbrad/UKBinCollectionData/wiki/Councils#google-calendar-public
